internal/cmd/flags.go

```diff
@@ -53,6 +56,70 @@ var (
 	}
 )
 
+var (
+	PflagKey = &pflag.Flag{
+		Name:      "key",
+		Shorthand: "k",
+		Usage:     "signing key name",
+	}
+	SetPflagKey = func(cmd *cobra.Command) {
+		cmd.Flags().StringP(PflagKey.Name, PflagKey.Shorthand, "", PflagKey.Usage)
+	}
+
+	PflagKeyFile = &pflag.Flag{
+		Name:  "key-file",
+		Usage: "signing key file",
+	}
+	SetPflagKeyFile = func(cmd *cobra.Command) {
+		cmd.Flags().String(PflagKeyFile.Name, "", PflagKeyFile.Usage)
+	}
+
+	PflagCertFile = &pflag.Flag{
+		Name:  "cert-file",
+		Usage: "signing certificate file",
+	}
+	SetPflagCertFile = func(cmd *cobra.Command) {
+		cmd.Flags().String(PflagCertFile.Name, "", PflagCertFile.Usage)
+	}
+
+	PflagTimestamp = &pflag.Flag{
+		Name:      "timestamp",
+		Shorthand: "t",
+		Usage:     "timestamp the signed signature via the remote TSA",
+	}
+	SetPflagTimestamp = func(cmd *cobra.Command) {
+		cmd.Flags().StringP(PflagTimestamp.Name, PflagTimestamp.Shorthand, "", PflagTimestamp.Usage)
+	}
+
+	PflagExpiry = &pflag.Flag{
+		Name:      "expiry",
+		Shorthand: "e",
+		Usage:     "expire duration",
+	}
+	SetPflagExpiry = func(cmd *cobra.Command) {
+		cmd.Flags().DurationP(PflagExpiry.Name, PflagExpiry.Shorthand, time.Duration(0), PflagExpiry.Usage)
+	}
+
+	PflagReference = &pflag.Flag{
+		Name:      "reference",
+		Shorthand: "r",
+		Usage:     "original reference",
+	}
+	SetPflagReference = func(cmd *cobra.Command) {
+		cmd.Flags().StringP(PflagReference.Name, PflagReference.Shorthand, "", PflagReference.Usage)
+	}
+
+	// TODO: cobra does not support string shorthand
+	PflagPluginConfig = &pflag.Flag{
+		Name:      "pluginConfig",
+		Shorthand: "c",
```

Comment:
Can you check with @SteveLasker that "c" is fine in this case?

Reply:
Ok, I will check it.

Reply:
> Can you check with @SteveLasker that "c" is fine in this case?

I have checked this with @shizhMSFT , we can drop the support.